### PR TITLE
More documentaries 

### PR DESCRIPTION
**Pull request by aungthurhahein**
Hi,
I added the list with some documentaries from my gibbon [playlist](https://gibbon.co/atrx/computer-science-documentaries).

### CODE REVIEW

**Comment by watson** [comment]
@aungthurhahein thanks for the many links :smiley: In the future, please make sure you only submit **one** link per pull request.

File: README.md [hunk 1]
```diff
@@ -39,21 +39,30 @@ A curated list of computer history videos, documentaries and related folklore ma
...
 - [Code Rush](https://www.youtube.com/watch?v=4Q7FTjhvZ7Y) (2000) - The story of Netscape and the birth of Mozilla <sup>[7.3/10](http://www.imdb.com/title/tt0499004/)</sup>
+- [The Code](https://www.youtube.com/watch?v=XMm0HsmOTFI) (2001) - The Histroy of Linux & FOSS <sup>[7.6/10](http://www.imdb.com/title/tt0315417/)</sup>
 - [Revolution OS](https://www.youtube.com/watch?v=jw8K460vx1c) (2001) - The story of GNU/linux and Open Source Movement <sup>[7.3/10](http://www.imdb.com/title/tt0308808/)</sup>
 - [The Secret History Of Hacking](https://www.youtube.com/watch?v=PUf1d-GuK0Q) (2001) - History of early hackers <sup>[7.5/10](http://www.imdb.com/title/tt2335921/)</sup>
 - [BBS The Documentary](https://www.youtube.com/playlist?list=PLgE-9Sxs2IBVgJkY-1ZMj0tIFxsJ-vOkv) (2005) - History of the Bulletin Board System <sup>[8.6/10](http://www.imdb.com/title/tt0460402/)</sup>
+- [Hackers Are People Too ](https://www.youtube.com/watch?v=7jciIsuEZWM) (2008) <sup>[6.1/10](http://www.imdb.com/title/tt1279942/)</sup>
```
Comment (watson) [inline README.md]: Pelase remove the trailing space inside the link after `Too`

File: README.md [hunk 2]
```diff
@@ -39,21 +39,30 @@ A curated list of computer history videos, documentaries and related folklore ma
...
 - [Something Ventured](http://www.imdb.com/title/tt1737747/) (2011) - The history of early venture capital <sup>[7.0/10](http://www.imdb.com/title/tt1737747/)</sup>
+- [The Codebreakers](https://www.youtube.com/watch?v=Zc-hlV2xbSg) (2011) - Documentary about how poor countries are using free and open source software (FOSS) for development purposes. <sup>[7.1/10](http://www.imdb.com/title/tt2119396/)</sup>
```
Comment (watson) [inline README.md]: Link movie linked to in the IMDb link isn't the right one. I couldn't find it on IMDb. The video it self links to this wikipedia article: https://en.wikipedia.org/wiki/The_Code-Breakers

According to the wiki article it's made in 2006. Please place it accordingly and just remove the IMDb link.

File: README.md [hunk 3]
```diff
@@ -39,21 +39,30 @@ A curated list of computer history videos, documentaries and related folklore ma
...
 ### Reflective interviews
 
+- [Steve Jobs - The Lost Interview](https://www.youtube.com/watch?v=TRZAJY23xio) (2012) <sup>[8.1/10](http://www.imdb.com/title/tt2104994/)</sup>
 - [The Great 202 Jailbreak](https://www.youtube.com/watch?v=CVxeuwlvf8w) (2013) - David Brailsford
 - [UNIX Special: Profs Kernighan & Brailsford](https://www.youtube.com/watch?v=vT_J6xc-Az0) (2015) - David Brailsford interviews Brian Kernighan
 
+
 ### Talks
 
+- [The Origins of Linux - Linus Torvalds](https://www.youtube.com/watch?v=WVTWCPoUt8w) (2009) - Linus Torvalds tells the story of how he went from writing code as a graduate student to become an icon for open source software.
```
Comment (watson) [inline README.md]: This talk was given in 2001, not 2009

**Comment by aungthurhahein** [comment]
Thx for checking my entries. 
Pls see  the last commit with your inputs.

**Comment by watson** [comment]
@aungthurhahein thanks :smile:

**Pull request merged**